src/Application.cpp

```diff
@@ -147,6 +148,7 @@ Application::Application(Settings &_settings, const Paths &paths,
     , logging(new Logging(_settings))
     , linkResolver(new LinkResolver)
     , streamerMode(new StreamerMode)
+    , pronouns(new pronouns::Pronouns)
```

Comment:
Nit: use `std::make_shared<...>` here ([it creates a single allocation at the cost of weak pointers keeping the object alive](https://devblogs.microsoft.com/oldnewthing/20230815-00)). Also makes it more clear that this is a shared_ptr instead of a unique_ptr.